queries.sql

```diff
@@ -31,3 +31,11 @@ select avg(weight_kg) from animals;
 select neutered, max(escape_attempts) as max_escape_attempts from animals group by neutered;
 select species, avg(escape_attempts) as avg_escape_attempts from animals where date_of_birth between '1990-01-01' and '2000-12-31' group by species;
 select species, min(weight_kg) AS min_weight, max(weight_kg) as max_weight from animals group by species;
+
+SELECT a.name FROM animals a JOIN owners o ON a.owner_id = o.id WHERE o.full_name = 'Melody Pond';
+SELECT a.name FROM animals a JOIN species s ON a.species_id = s.id WHERE s.name = 'Pokemon';
```

Comment:
- [ ] In these two queries you should select all the data not only names, kindly consider using the `*` to select all the data instead

Reply:
![image](https://github.com/ShaliniNayan/Vet-clinic/assets/117961613/94898014-a914-4486-82a7-94b9be094313)